pyscf/adc/uadc.py

```diff
@@ -4142,10 +4142,10 @@ def analyze_eigenvector_ea(adc):
     f_bbb = s_bbb + n_doubles_bbb
 
     U = adc.U
-
+    print("eig vec shape: ", U.shape)
     for I in range(U.shape[1]):
-        U1 = U[I, :f_b]
-        U2 = U[I, f_b:]
+        U1 = U[:f_b, I]
```

Comment:
I thought this was addressed by @Samragni-93. I am surprised to these changes once again.